cmd/indexer/db/migrations/000003_transactions.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS transactions(
+    -- Primary Key
+    protocol        SMALLINT NOT NULL,
+    network         SMALLINT NOT NULL,
+    hash            BYTEA NOT NULL,
+    -- Values
+    tx_from            BYTEA NOT NULL,
```

Comment:
Yes I see your point about `from` I think something like `[from]` in the query would work but not sure if `squirrel` supports this and the `tx_` prefix is ok too.  I think best to keep the prefix.